classes/patterns/class-issues.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * Issues class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Issues pattern includes:
+ * Column, Image, Heading, Paragraph, Media & Text.
+ *
+ * @package P4GBKS\Patterns
+ */
+class Issues extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/issues';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 */
+	public static function get_config(): array {
+		return [
+			'title'      => __( 'Issues', 'planet4-blocks-backend' ),
+			'keywords'   => [ 'column', 'image', 'heading', 'paragraph', 'media', 'text' ],
+			'categories' => [ 'planet4' ],
+			'content'    => '
+				<!-- wp:group {"backgroundColor":"grey-10","textColor":"grey-80","className":"block-wide"} -->
+				<div class="wp-block-group block-wide has-grey-80-color has-grey-10-background-color has-text-color has-background"><!-- wp:group {"className":"container"} -->
+				<div class="wp-block-group container"><!-- wp:spacer {"height":"60px"} -->
+				<div style="height:60px" aria-hidden="true" class="wp-block-spacer"></div>
+				<!-- /wp:spacer -->
+
+				<!-- wp:heading {"textAlign":"center","level":1,"textColor":"grey-80", "placeholder":"' . __( 'Enter title', 'planet4-blocks-backend' ) . '"} -->
+				<h1 class="has-text-align-center has-grey-80-color has-text-color"></h1>
+				<!-- /wp:heading -->
+
+				<!-- wp:paragraph {"align":"center", "placeholder":"' . __( 'Enter description', 'planet4-blocks-backend' ) . '"} -->
+				<p class="has-text-align-center"></p>
+				<!-- /wp:paragraph -->
+
+				<!-- wp:spacer {"height":"37px"} -->
+				<div style="height:37px" aria-hidden="true" class="wp-block-spacer"></div>
+				<!-- /wp:spacer -->
+
+				<!-- wp:group {"layout":{"contentSize":"100%"}} -->
+				<div class="wp-block-group">
+
+				<!-- wp:columns -->
+				<div class="wp-block-columns">
+
+				<!-- wp:column {"verticalAlignment":"center","width":"25%","backgroundColor":"white"} -->
+				<div class="wp-block-column is-vertically-aligned-center has-white-background-color has-background" style="flex-basis:25%"><!-- wp:group {"layout":{"wideSize":"90%","contentSize":"90%"}} -->
+				<div class="wp-block-group"><!-- wp:spacer {"height":"16px"} -->
+				<div style="height:16px" aria-hidden="true" class="wp-block-spacer"></div>
+				<!-- /wp:spacer -->
+				<!-- wp:media-text {"mediaWidth":20,"isStackedOnMobile":false,"imageFill":false} -->
+				<div class="wp-block-media-text alignwide" style="grid-template-columns:20% auto"><figure class="wp-block-media-text__media"></figure><div class="wp-block-media-text__content"><!-- wp:paragraph {"align":"left","placeholder":"' . __( 'Enter text', 'planet4-blocks-backend' ) . '","style":{"typography":{"fontSize":"17px","fontStyle":"normal","fontWeight":"700"}},"textColor":"grey-80"} -->
```

Comment:
Also, maybe you don't know this (I sure didn't 😬) but you can actually use `rem` values instead of `px` for the `fontSize` attribute, I'm not sure about this specific case but it might be better?